lib/index_group.rb

```diff
@@ -15,6 +15,9 @@ class IndexGroup
     def initialize(base_uri, name, schema, search_config)
       @base_uri = base_uri
       @client = Services::elasticsearch(hosts: base_uri)
+      # Index creation can take longer than other requests, so create a separate
+      # client with a longer timeout
+      @index_creation_client = Services::elasticsearch(hosts: base_uri, timeout: 10)
```

Comment:
@MatMoore and I had a chat about this. We think that it's safe to do this everywhere because we don't create indexes very often (I think it's just the overnight bulk import) and it's OK if that takes longer than a typical fast request which fetches or updates data.